frontend-react/jest.config.ts

```diff
@@ -51,6 +51,7 @@ const config: Config.InitialOptions = {
         "^.*\\.mdx": "<rootDir>/src/__mocks__/mdxFrontmatterMock.tsx",
         "^react-helmet-async$":
             "<rootDir>/src/__mocks__/reactHelmetAsyncMock.tsx",
+        "^uuid$": require.resolve("uuid"),
```

Comment:
We'll need to see what needs to be done (if anything) for vitest when you update this PR.